addons/website_blog/views/website_blog_templates.xml

```diff
@@ -161,7 +161,7 @@
 
                             <!-- To display tags //no options -->
                             <t t-foreach="blog_post.tag_ids" t-as="one_tag">
-                                <a class="mr8" t-attf-href="#{blog_url(tag=one_tag.id, date_begin=False, date_end=False)}" t-esc="one_tag.name"/>
+                                <a class="mr8" t-attf-href="#{blog_url(tag=slug(one_tag), date_begin=False, date_end=False)}" t-esc="one_tag.name"/>
```

Comment:
@JKE-be Your fix is not fixing this deduplication of URLs.

However, I'm realizing this fix ain't perfect too :thinking:. When the user (and, worse, the crawler) switches the language while seeing 1 tag, the tag slug appears in the prior language still.

This should be then redirected with a 301 by the controller, don't you think? Just as it always happens in ids-vs-slugs across odoo website.